docs/maui/views/RatingView.md

```diff
@@ -0,0 +1,519 @@
+---
+title: RatingView - .NET MAUI Community Toolkit
+author: GeorgeLeithead
+description: "The `RatingView` is designed to provide developers with a flexible and customizable rating mechanism, similar to those used on popular review and feedback platforms."
+ms.date: 09/19/2024
+---
+
+# RatingView
+
+The CommunityToolKit MAUI `RatingView` is a control designed to provide developers with a flexible and customizable rating mechanism, similar to those used on popular review and feedback platforms.
+
+
+## Syntax
+
+### Including the XAML namespace
+
+[!INCLUDE [XAML usage guidance](../includes/xaml-usage.md)]
+
+### Using the RatingView
+
+The following example shows how to create a `RatingView`:
+
+![Screenshot of an RatingView example](../images/views/RatingView.png "RatingView example")
+
+```xaml
+<ContentPage
+	x:Class="CommunityToolkit.Maui.Sample.Pages.MyPage"
+	xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+	xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+	xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit">
+	<VerticalStackLayout>
+		<toolkit:RatingView
+			EmptyColor="White"
+			FilledColor="Blue"
+			IsReadOnly="False"
+			ItemPadding="3,7,7,3"
+			ItemShapeSize="37"
+			MaximumRating="5"
+			RatingFill="Shape"
+			Rating="4.5"
+			Shape="Star"
+			ShapeBorderColor="Red"
+			ShapeBorderThickness="1"
+			Spacing="3" />
+	</VerticalStackLayout>
+</ContentPage>
+```
+
+The equivalent C# code is:
+
+```csharp
+using CommunityToolkit.Maui.Views;
+
+partial class MyPage : ContentPage
+{
+	public MyPage()
+	{
+		RatingView ratingView = new()
+		{
+			EmptyColor = Colors.White,
+			FilledColor = Colors.Blue,
+			IsReadOnly = false,
+			ItemPadding = new Thickness(3,7,7,3),
+			ItemShapeSize = 37,
+			MaximumRating = 5,
+			RatingFill = RatingFillElement.Shape,
+			Rating = 4.5,
+			Shape = RatingViewShape.Star,
+			ShapeBorderColor = Colors.Red,
+			ShapeBorderThickness = 1,
+			Spacing = 3,
+		};
+
+		Content = ratingView;
+	}
+}
+```
+
+## Properties
+
+| Property | Type | Description |
```

Comment:
I think we should only include the properties that we are providing and not those provided by base classes coming out of .NET MAUI. We can link to any MAUI documentation to cover those properties if you want though

Reply:
Removed base class properties and provided appropriate link.